recipe/patches/0001-loosen-requirements.patch

```diff
@@ -1,41 +1,27 @@
-From 2b14b3c38210f8454894cec887915820a28b3ef8 Mon Sep 17 00:00:00 2001
-From: "Uwe L. Korn" <uwe.korn@quantco.com>
-Date: Thu, 30 Dec 2021 06:57:09 +0200
-Subject: [PATCH] loosen requirements
-
-* Remove unused libclang requirement
-* Remove unecessary tensorflow-io-gcs-filesystem
-
-Co-authored-by: Mark Harfouche <mark.harfouche@gmail.com>
-Co-authored-by: ngam
----
```

Comment:
Please keep the patch in the format of `git format-patch` and keep attribution. :)